### ISSUE TITLE
Enjoyhint with angular6

### ISSUE THREAD

**Issue by ashishupadhyay** [xbsoftware/enjoyhint#101]
Hello,

I really liked this library and i was able to make it work with angular v1.x but we are moving to angular 6. I tried to download enjoyhint from npm but it has the old version. Has anyone implemented latest version of enjoyhint on angular 2 or above. Would be really helpful if you can tell me how you did it.

Thanks

**Comment by rhlkumar234** [comment]
@ashishupadhyay were you able to make it work without jquery for angular v1.x?

**Comment by ashishupadhyay** [comment]
Yeah I was able to make it work with angularjs 1.6.

**Comment by rhlkumar234** [comment]
could you please elaborate? Did you create a directive?

**Comment by ashishupadhyay** [comment]
No, I just had to show a demo so I used jQuery within my angularjs. Eventually I had to use it in angular 6 but wasn't able to do so. Latest enjoyhint version is not available on npm.

**Comment by tsemashkoXB** [comment]
Thanks for reporting this, and sorry for the long silence.

This is fixed in the new **v5** release. EnjoyHint is now a modern TypeScript package (no jQuery runtime) with proper ESM/CJS exports, so it works with Angular and other modern bundlers out of the box.

Please try the latest version and let us know if anything still doesn’t work for you.
